db/migrate/20190606194808_change_default_user_role_back.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDefaultUserRoleBack < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default(:users, :role, from: 1, to: 0)
```

Comment:
why r u changing default of the same column in 2 different migrations in the same PR?

Reply:
So can i simply delete them? I just thought its not a good practice to delete existing migrations

Reply:
what do u mean by "existing"? as i see the `db/migrate/20190605150126_change_default_user_role.rb` is added in this PR